### PR TITLE
Updated the README.md to describe the repo.

### PR DESCRIPTION
**Pull request by Sarahhud**

### CODE REVIEW

**Comment by NicholasCobbFromGoogle** [comment]
LGTM

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -7,3 +7,9 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.image{
+  border: 10px solid rgb(122, 6, 30);
+  float: inherit;
+  color:
```
Comment (NicholasCobbFromGoogle) [inline portfolio/src/main/webapp/style.css:14]: I think that this "color" line is not needed.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,9 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.image{
```
Comment (NicholasCobbFromGoogle) [inline portfolio/src/main/webapp/style.css:11]: nit: Please add a space between the class name and the bracket
 (".image {")

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,10 +8,20 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Sarah Hudson's Portfolio</h1>
+    <img src="/images/Sarah.jpg" class="image" alt="Sarah Hudson's profile photo." width="500" height="500">
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/index.html:12]: Nit: Make img tag have same indentation as surrounding tags.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ["My birthday is June 4th.", "I have a website where I post things I am learning about CS.", "I have two sisters.", "I have a border collie named Scout."];
```
Comment (mbehrlich) [inline portfolio/src/main/webapp/script.js:20]: Very long line. Please break up by item.

For instance:

const greetings = [
  "My birthday is...",
  "...",
  "...",
];

**Pull request merged**